index.html

```diff
@@ -0,0 +1,177 @@
+        </div>
+        <div class="container">
+
+          <div class="card1 cards">
+            <i class="fa fa-desktop screen-icon" aria-hidden="true"></i>
+            <h3>Lecture</h3>
+            <p>Listen to the speakers from various countries about the Creating and selling NFTs</p>
+          </div>
+
+          <div class="card2 cards">
+            <i class="fa-solid fa-coins"></i>
+            <h3>NFT Exhibition</h3>
+            <p>Explore different NFTs already trending in the global market</p>
+          </div>
+
+          <div class="card3 cards">
+            <i class="fa-solid fa-comment-dollar"></i>
+            <h3>Forum</h3>
+            <p>Have the time to share your thoughts and opinions with experts for each topic</p>
+          </div>
+
+          <div class="card4 cards">
+            <i class="fa-solid fa-chalkboard-user"></i>
+            <h3>Workshop</h3>
+            <p>try creating your own NFTs using our various open-end frameworks available</p>
+          </div>
+
+          <div class="card5 cards">
+            <i class="fa-solid fa-link"></i>
+            <h3>NFT Initate</h3>
+            <p>Get opportunities to link up and network with NFT traders and High class buyers all around the world,</p>
+          </div>
+
+        </div>
+        <h2 class="program-text desktop-only">View the Entire Program</h2>
+        <button type = "submit" class="program-text mobile-only">Join NFT Summit 2022</button>
+        
+      </section>
+
+      <!-- The Speakers Section -->
+      <div id="speaker-section"></div>
+     
+
+      <!-- The list of Partners Section -->
+      <section id="Sponsor">
+        <h3>Our Sponsors and Partners</h3>
+        <div class="line"> </div>
+        <div class="icons">
+          <ul>
+            <li class="lists">
+              <a href="binance.com"><img src="./images/binance-Logo.png" class="partner-logo" alt="Binance"></a>
+
+              <a href="Coinbase.com"><img src="./images/coinbase.svg" class="partner-logo" alt="CoinBase"></a>
+
+              <a href="Flow.com"><img src="./images/Flow.png" class="partner-logo" alt="flow"></a>
+
+              <a href="moonpay.com"><img src="./images/moonpaymain.svg" class="partner-logo" alt="MoonPay"></a>
+            </li>
+            
+          </ul>
+        </div>
+      </section>
+
+      <!-- The Footer for the Home Page -->
+      <footer class="footer">
+        <div class="logo-icon">
+          <div><a href="index.html" class="nft-logo"><img src="./images/nft-logo.png" class="nft-logo" alt="nft logo"></a>
+          </div>
+          <h5 class="logo-text">NFT GLOBAL <br> SUMMIT 2022</h5>
+        </div>
+        
+        <div class="footer-text">
+          <h6>2022 - NFT GLOBAL SUMMIT, <br> All Rights Reserved</h6>
+          <small class="footer-paragraph">Website designed and built by Promise Okechukwu. Contact:+234 701 289 7856 <br> UI by Cindy Shin</small>
+        </div>
+      </footer>
```

Comment:
- [OPTIONAL] According to the original design, the `partners` and `footer` sections should not be shown in the **mobile version**, the last section for the `mobile version` should be the speakers (or the artists in your project). Please, kindly address this issue in the mobile version in order to match the design :smiley: :raised_hands:

| Original design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/147131660-823d562f-6346-4262-9854-f7292853388d.png) | ![image](https://user-images.githubusercontent.com/45231276/181242008-f29ab373-601a-4acb-a7c7-54d826279cc1.png) |